charts/gardener-extension-provider-vsphere/templates/deployment.yaml

```diff
@@ -44,6 +44,7 @@ spec:
         - --webhook-config-server-port={{ .Values.webhookConfig.serverPort }}
         - --disable-controllers={{ .Values.disableControllers | join "," }}
         - --disable-webhooks={{ .Values.disableWebhooks | join "," }}
+        - --health-bind-address=:{{ .Values.healthPort }}
```

Comment:
It makes sense to wrap this in case the values are not provided, see https://github.com/gardener/gardener/blob/e8e73842ad0acfe30c02eb7d17c8c7b2fd42a4e5/charts/gardener/gardener-resource-manager/templates/deployment.yaml#L62-L64